### DISCUSSION TITLE
Temas para el Obligatorio

### DISCUSSION THREAD

**Discussion by agusRobaina** [IngenieriaSoftwareTeam/obligatorio1-Nicolas-Trom-Joaquin-Rega-Agustin-Robaina#3] [Ideas]
Idea 1 :  
- Proyecto que le permita al usuario visualizar las recetas disponibles en el sistema y escoger una para cocinar.
- Le permitirá al usuario seleccionar los alimentos que dese incluir en su dieta para que se le recomiende recetas que los incluya
- Le permitrá al usuario fijar un objetivo y trackear el progreso del mismo

**Comment by JoaquinRega** [comment]
Idea 2: Aplicacion para estudiantes, en donde puede gestionar proyectos, parciales y todo lo que tenga q ver con sus estudios

**Comment by Nicolas-Trom** [comment]
Idea 3 : Aplicacion de Resultados de Futbol, en donde se puede visualizar resultados de partidos y estadisticas tanto del equipo como de jugadores.
